From 3193b1159d1579fa56dd6aa5ec58c025228626e7 Mon Sep 17 00:00:00 2001
From: Josh Watzman <jwatzman@fb.com>
Date: Tue, 20 Sep 2016 02:05:11 -0700
Subject: [PATCH] Remove warn_error_A from OSS build

Summary: IMO this is causing more trouble than good -- with all the different versions of OCaml floating around on different OSes, and with our fairly poor CI coverage of them, we tend to have some warning somewhere that fails the build for some people for no good reason. (4.03 on OS X in particular continues to cause me grief.) I don't think we get enough PRs to justify this.

Reviewed By: dabek

Differential Revision: D3884617

fbshipit-source-id: d963f1b001b979bcd6a995bf41074f9fb429697d
---
 hphp/hack/src/_tags | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hphp/hack/src/_tags b/hphp/hack/src/_tags
index 800a0aa..b2da669 100644
--- a/hphp/hack/src/_tags
+++ b/hphp/hack/src/_tags
@@ -1,4 +1,4 @@
-<**/*.ml*>: ocaml, warn_A, warn(-3-4-6-29-35-44-48-50-52), warn_error_A
+<**/*.ml*>: ocaml, warn_A, warn(-3-4-6-29-35-44-48-50-52)
 <*.ml*>: warn(-27)
 <client/*.ml*>: warn(-27)
 <deps/*.ml*>: warn(-27)
